### PR TITLE
Added classes for DisplacementSensor and HeadingSensor

### PR DESCRIPTION
**Pull request by vincent-miller**
Added two more empty classes in which to implement reading information from the mouse and heading sensor. This data will be integrated in PositionTracking.

### CODE REVIEW

**Comment by celskeggs** [comment]
I'll merge this once it looks good, but it's probably not worth adding pull requests for small additions if no one else is depending on them to do their work.

File: QuasarHelios2015/src/org/team1540/quasarhelios/DisplacementSensor.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package org.team1540.quasarhelios;
+
+import java.awt.geom.Point2D;
```
Comment (celskeggs) [inline QuasarHelios2015/src/org/team1540/quasarhelios/DisplacementSensor.java]: Please don't use AWT unless you're doing graphics in your code somewhere. I'd rather you write your own Point2D than depend on functionality from Java's graphics library.

**Comment by celskeggs** [comment]
To clarify, you should still send everything through code review, but don't send it until you have something significant to be reviewed.

**Comment by vincent-miller** [comment]
I've removed the AWT dependencies and implemented basic integration. The integration method may have to be improved (for example if we want to include the headings at the beginning and end of a time interval).

**Comment by vincent-miller** [comment]
Should I change PositionTracking.getPosition to use CCRE FloatInputs instead (replace it with getXChannel and getYChannel)?

**Comment by vincent-miller** [comment]
When I poll the mouse, the return is displacement since the last event, not velocity. This means that it is time-independent. The getDisplacement method would block until a new mouse event is available (unless you think it shouldn't be) so that would serve in place of a time delay.

**Comment by celskeggs** [comment]
Changing getPosition to use that sounds useful.

That makes more sense with `getDisplacement`. Perhaps you should rename it to make it clear what it does? (And perhaps reorder them so that it gets the heading after each displacement, not before.)

**Comment by celskeggs** [comment]
We've canceled this. Oh well.

**Pull request closed without merging**